src/null-safety/index.md

```diff
@@ -115,6 +115,16 @@ To resolve these issues, check for
 of any packages you installed from pub.dev, and [migrate](#migrate) all
 of your own source code to use sound null safety.
 
+As of late January 2023, Dart 3 alpha is available from the Dart dev channel
+and the Flutter master channel, and we recommend you test your code
+for Dart 3 compatibility using that release:
+
+```terminal
+$ dart --version                     # make sure this reports 3.0.0-151.0.dev or higher
+$ dart pub get                       # this should resolve without issues
+$ dart analyze / flutter analyze     # this should pass without errors
```

Comment:
Kind of the same as the previous comment, "this _should_ ....": if it should do something, can we point them to some troubleshooting or something to do if it _doesn't_ do what it should? 

If the first comment I left about pointing them to how to set up those channels would ensure that these lines will behave as expected, then nothing else needs to be added here.